site/en/docs/workbox/index.md

```diff
@@ -6,7 +6,7 @@ description: 'Tooling! Workbox! Stuff!'
 
 <!-- TODO(samthor): Using a h2 here doesn't match our designs, but it's a linter requirement -->
 
-## Case Studies
+##  heading for the linter (see todo)
```

Comment:
We can actually empty this file out if you just want to clear the content for now. The linter requirement is only _if_ there's any content, and we can probably disable it.

However, the design that's been shown to me basically has a couple of "Featured" cards here. I was just thinking to push this into pure HTML inserted here, rather than trying to design a new solution, since it's (a) just for Workbox and (b) going to be fairly static.